app/config/publishing.js

```diff
@@ -0,0 +1,19 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  dataPublishingMaxBatchSize: Joi.number().default(250)
```

Comment:
Checking the code, overall maximum batch size in some cases would in multiple of two of the figure provided. I can see that it would be difficult to distribute a fixed overall maximum batch size between/among different data types, hence your reason for allocating maximum batch size to individual data-type, can we let the name here reflect that?

Reply:
Yup, agree -- changing